platform/security/sts/security-sts-server/src/main/resources/OSGI-INF/blueprint/cxf-sts.xml

```diff
@@ -258,7 +258,7 @@
         <argument ref="STSProperties"/>
         <cm:managed-properties persistent-id="ddf.security.sts"
                                update-strategy="container-managed"/>
-        <property name="issuer" value="${org.codice.ddf.system.hostname}"/>
+        <property name="issuer" value="https://${org.codice.ddf.system.hostname}:${org.codice.ddf.system.httpsPort}/services/idp/login"/>
```

Comment:
If you want `services` could also be replaced with `${org.codice.ddf.system.rootContext}` if you wanted.

Reply:
Yeah it is set via a config, so probably a good thing to do.